datadict/datadict-sqlite.inc.php

```diff
@@ -84,12 +84,16 @@ function actualType($meta)
 	}
 
 	// return string must begin with space
-	function _createSuffix($fname, &$ftype, $fnotnull, $fdefault, $fautoinc, $fconstraint, $funsigned)
+	function _createSuffix($fname,&$ftype,$fnotnull,$fdefault,$fautoinc,$fconstraint,$funsigned,$fprimary,&$pkey)
```

Comment:
Please keep the space after comma (and if you don't mind, add it to the overrides in the inherited methods)

Reply:
I tried to mimic the existing format, but I also like spaces.